build/requirements.txt

```diff
@@ -10,6 +10,7 @@ thinc==8.1.0
 numba==0.60.0
 wrapit==0.3.0
 h5py==3.12.1
+gitpython
```

Comment:
add a specific version to prevent potential future breaks due to compatability

Reply:
Done. I've set versions for all deps without versions